### PR TITLE
Issue 1 capitalize commands part 1

### PR DESCRIPTION
**Pull request by rqho**

### CODE REVIEW

File: src/main/java/frc/robot/commands/drivetrain/DriveStraight.java [hunk 1]
```diff
@@ -3,36 +3,35 @@
...
 import edu.wpi.first.wpilibj2.command.PIDCommand;
 import frc.robot.Robot;
-import frc.robot.subsystems.drivetrain;
+import frc.robot.subsystems.DriveTrain;
 
-public class driveStraight extends PIDCommand {
-    
-    private final drivetrain m_drive;
+public class DriveStraight extends PIDCommand {
+    private final DriveTrain drive;
```
Comment (alantango) [inline src/main/java/frc/robot/commands/drivetrain/DriveStraight.java]: Good. As long as we use a consistent style. Some like naming member variables with a `m_` prefix, this way is good too. Often we just name the variable as the class name, with first character turned lowercase. Whichever way, consistency is key.

File: src/main/java/frc/robot/commands/drivetrain/DriveToApril.java [hunk 2]
```diff
@@ -2,12 +2,11 @@
...
 import edu.wpi.first.wpilibj2.command.PIDCommand;
-import frc.robot.subsystems.drivetrain;
+import frc.robot.subsystems.DriveTrain;
 import frc.robot.Robot;
 
-public class driveToApril extends PIDCommand {
-    
-    public driveToApril(drivetrain m_drive) {
+public class DriveToApril extends PIDCommand {
+    public DriveToApril(DriveTrain m_drive) {
```
Comment (alantango) [inline src/main/java/frc/robot/commands/drivetrain/DriveToApril.java]: Now do you want to rename this parameter `m_drive` to be consistent as above, too? If so, think what would be a proper variable name.

File: src/main/java/frc/robot/Main.java [hunk 3]
```diff
@@ -7,19 +7,20 @@
...
- * Do NOT add any static variables to this class, or any initialization at all. Unless you know what
- * you are doing, do not modify this file except to change the parameter class to the startRobot
- * call.
+ * Do NOT add any static variables to this class, or any initialization at all.
+ * Unless you know what you are doing, do not modify this file except to change 
+ * the parameter class to the startRobot call.
  */
 public final class Main {
-  private Main() {}
+    private Main() {
```
Comment (alantango) [inline src/main/java/frc/robot/Main.java:15]: Sensible update.

File: src/main/java/frc/robot/commands/drivetrain/DriveToReflective.java [hunk 4]
```diff
@@ -2,12 +2,11 @@
...
 import edu.wpi.first.wpilibj2.command.PIDCommand;
-import frc.robot.subsystems.drivetrain;
+import frc.robot.subsystems.DriveTrain;
 import frc.robot.Robot;
 
-public class driveToReflective extends PIDCommand {
-    
-    public driveToReflective(drivetrain m_drive) {
+public class DriveToReflective extends PIDCommand {
+    public DriveToReflective(DriveTrain m_drive) {
```
Comment (alantango) [inline src/main/java/frc/robot/commands/drivetrain/DriveToReflective.java]: Another `m_drive`....

File: src/main/java/frc/robot/commands/drivetrain/DriveTrackApril.java [hunk 5]
```diff
@@ -2,18 +2,17 @@
...
-    public driveTrackApril(drivetrain m_drive) {
+public class DriveTrackApril extends PIDCommand {
+    public DriveTrackApril(DriveTrain drive) {
         super(new PIDController(0.0075, 0.005, 0),
         
             () -> Robot.photon.getYaw(),
             12,
-            (output) -> m_drive.drive(0, output),
-            m_drive);
+            (output) -> drive.drive(0, output),
```
Comment (alantango) [inline src/main/java/frc/robot/commands/drivetrain/DriveTrackApril.java]: Here's what I was hinting about in previous comments. "drive.drive"...? Would it be better to be `driveTrain.drive`?

File: src/main/java/frc/robot/commands/drivetrain/DriveTrackReflective.java [hunk 6]
```diff
@@ -2,12 +2,11 @@
...
 import edu.wpi.first.wpilibj2.command.PIDCommand;
-import frc.robot.subsystems.drivetrain;
+import frc.robot.subsystems.DriveTrain;
 import frc.robot.Robot;
 
-public class driveTrackReflective extends PIDCommand {
-    
-    public driveTrackReflective(drivetrain m_drive) {
+public class DriveTrackReflective extends PIDCommand {
+    public DriveTrackReflective(DriveTrain m_drive) {
```
Comment (alantango) [inline src/main/java/frc/robot/commands/drivetrain/DriveTrackReflective.java]: Probably last `m_drive`... :-)

File: src/main/java/frc/robot/commands/rgb/rainbow.java [hunk 7]
```diff
@@ -2,22 +2,24 @@
...
-        
+
     }
 
     @Override
     public void end(boolean interrupted) {
+
     }
-} 
+}
```
Comment (alantango) [inline src/main/java/frc/robot/commands/rgb/rainbow.java:25]: Generally we need a new line at the end of our code. As you can see Github even highlight it here (the red circle with a minus sign inside). So just add a new line at the end.

File: src/main/java/frc/robot/commands/rgb/red.java [hunk 8]
```diff
@@ -18,5 +18,6 @@ public void execute() {
 
     @Override
     public void end(boolean interrupted) {
+        
     }
-} 
+}
```
Comment (alantango) [inline src/main/java/frc/robot/commands/rgb/red.java:23]: Missing new line.

File: src/main/java/frc/robot/commands/rgb/rainbow.java (hunk 7 above)
Comment (rqho) [reply src/main/java/frc/robot/commands/rgb/rainbow.java:25]: not in issue (a-e)

File: src/main/java/frc/robot/commands/rgb/red.java (hunk 8 above)
Comment (rqho) [reply src/main/java/frc/robot/commands/rgb/red.java:23]: not in issue (a-e)

File: src/main/java/frc/robot/commands/rgb/rainbow.java (hunk 7 above)
Comment (alantango) [reply src/main/java/frc/robot/commands/rgb/rainbow.java:25]: Oops, I missed. Ok then!

File: src/main/java/frc/robot/commands/rgb/red.java (hunk 8 above)
Comment (alantango) [reply src/main/java/frc/robot/commands/rgb/red.java:23]: oops, ok.

**Review by alantango — APPROVED** [review]
LGTM

**Pull request merged**